SpatialGDK/Documentation/content/get-started/example-project/exampleproject-cloud-deployment.md

```diff
@@ -35,190 +37,274 @@ When you signed up for SpatialOS, your account was automatically associated with
+
   <%(/Expandable)%>
-  Then, back in the Editor Deploy window:</br></br>
-  * **Add simulated players**: Check the box.</br></br>
-  * **Deployment Name**: Enter a name for your simulated player deployment. This labels the deployment in the Console. Make it different to your game deployment name.</br></br>
-  * **Number of Simulated Players**: Choose the number of simulated players you want to start. </br></br>
-  * **Region**: The real-world geographical location that you want your simulated players cloud deployment hosted in. 
+
+    Back in the Editor Deploy window:</br></br>
+
+    * **Add simulated players**: Check the box.
+    </br></br>
+
+    * **Deployment Name**: Enter a name for your simulated player deployment. This labels the deployment in the Console. Make   itdifferent to your game deployment name.
+    </br></br>
+
+    * **Number of Simulated Players**: Choose the number of simulated players you want to start. 
+    </br></br>
+
+    * **Region**: The real-world geographical location that you want your simulated players cloud deployment hosted in. 
+    </br></br>
+
+4. Select **Launch deployment**.
+  Your deployment(s) won’t launch instantly - the Editor launches a console window which shows the progress of the deployment. When your deployment(s) have launched, you can open the Console at [console.improbable.io](https://console.improbable.io/) to view them.
+
+> **Tip:** You can set default values for all the fields in the cloud deployment window, using the Cloud section of the [SpatialOS Editor Settings panel]({{urlRoot}}/content/unreal-editor-interface/editor-settings).
+
+<%(#Expandable title="What's the launch configuration file?")%>
+
+This `.json` file contains the configuration parameters for starting a deployment, including:
+* the game template - defines the compute resources your deployment needs (see the documentation on [game templates andpricing](https://docs.improbable.io/reference/latest/shared/pricing-and-supportpricing-details#introduction-to-game-templates)).
+* `dimensionsInWorldUnits` - defines the size of your SpatialOS game world in X and Y.
+* worker types - lists the worker type names you have set up for your project. For this project; an Unreal server-worker `UnrealWorker` and an Unreal client-worker `UnrealClient`.
+This project comes with a ready-made launch configuration file `one_worker_test.json`. Note that you can call your launch  configuration file any name you choose.
+
+You can find out more in the SpatialOS Worker SDK documentation: [launch confguration file](https://docs.improbable.io/reference/14.0/shared/flexible-project-layout/reference/launch-configuration) (but note that this may contain details on  parameters not relevant to Unreal-developed projects.)
```

Comment:
You can find out more in the SpatialOS Worker SDK documentation about the launch configuration file. Note that this may contain additional details on parameters that are not relevant to projects based on Unreal

Or something??